src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -192,7 +197,8 @@
      * The endpoint will try immediately to connect to the central message broker and tries automatically to maintain
      * this connection in the background.
      *
-     * @param uuid                              Unique ID of the endpoint.
+     * @param uuid                              Unique ID of the endpoint or a filename if the {filename}.properties file
```

Comment:
- Rename Argument to something like `uuidOrApplicationName`
- Update documentation for all constructors